modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase

```diff
@@ -588,9 +588,13 @@ definition {
 			ObjectAdmin.gotoRelationshipsTab();
 
 			ObjectCustomLayouts.addEntryOnRelationshipTab(selectEntry = "Entry Test B");
+
+			Alert.viewSuccessMessage();
 		}
 
 		task ("When delete the entry on Relationship Tab") {
+			ObjectPortlet.viewEntry(entry = "Entry Test B");
+
```

Comment:
Why are we adding this macro here? For context this is what the macro is doing:
```
		WaitForSPARefresh();

		SelectFrameTop();

		AssertElementPresent(
			key_entry = ${entry},
			locator1 = "ObjectPortlet#ENTRY_VALUE");
```

Reply:
Why not use `WaitForElementPresent` instead? In general, we should avoid making assertions outside of `Then`

Reply:
@timpak 
Actually the `WaitForElementPresent` was my first attempt.
How we can see in [Testray](https://testray.liferay.com/home/-/testray/case_results/2266288475?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayCaseName%3DCanDeleteNewEntryAddedOnRelationshipTab%26testrayBuildId%3D2266283779%26testrayRunId%3D2266283830%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)

Reply:
Hmm did you try using `ObjectPortlet#ENTRY_VALUE` instead of `ObjectAdmin#VIEW_OBJECT_FIELD_LABEL`?

Reply:
With `ObjectAdmin#VIEW_OBJECT_FIELD_LABEL` with `WaitForElementPresent` the test has passed locally but fallen on [ci](https://testray.liferay.com/reports/production/logs/2023-05/test-1-38/test-portal-acceptance-pullrequest(master)/7489/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ObjectCustomLayouts_CanDeleteNewEntryAddedOnRelationshipTab/index.html.gz?authuser=0)